.storybook/preview.js

```diff
@@ -6,7 +6,7 @@ import { setupTwig } from './setupTwig';
 // GLOBAL CSS
 import '../components/style.scss';
 import '../components/00-base/colors/cl-colors.scss';
-import '../components/00-base/spacing/cl-spacing.scss';
+import '../components/00-base/base/cl-spacing.scss';
```

Comment:
Renaming `00-base` to `00-tokens` in the more recent PR is a good call as this seems a bit redundant. That being said, what else are you thinking would have a home in `00-tokens/base/`? I'm thinking this should be renamed to `00-tokens/breakpoints/`.

Reply:
I'll make this change in my current branch (Color Pairings) because other things have changed. That way I don't have to do the merge-conflict dance.

To answer your question, though, "I don't know." In the past, we've put things like breakpoints, mixins, and base styles like box-sizing in the `base` folder. Most of that won't be needed on this project since they'll come from the tokens in some form. For now, I've started putting the "base" styles (like the stuff below) in the main `style.scss` file along with the imports. We'll see if that needs to move (i.e. if it gets too big) later.

```
:root {
  --color-text: var(--color-basics-black);
}

html {
  box-sizing: border-box;
}

*,
*::before,
*::after {
  box-sizing: inherit;
}

body {
  font: var(--font-style-body-default);
}
```

Reply:
For those styles specifically I see why you would add them to `style.scss` and that makes sense to a certain degree I think. But given this project is more of a platform I could see these styles living in `00-tokens/00-reset/` or `00-tokens/00-foundation/` or something like that.